server/db_data_init.py

```diff
@@ -0,0 +1,198 @@
+from passlib.hash import pbkdf2_sha256
+from sqlalchemy import create_engine
+from sqlalchemy.sql.expression import insert
+
+from server.models import (Base, user, user_status, tag, token, event, event_history,
+                           event_status, event_tag, category, subscribe,
+                           feedback, gallery, role)
+
+
+engine = create_engine('postgresql://admin:1@localhost:5555/eventme',
+                        echo=True)
+Base.metadata.create_all(bind=engine)
+
+role_data = (
+    {"role": "admin"},
```

Comment:
You may instantiate model directly here: `role.Role(role="admin")`. But it's up to you.